run_benchmark.py

```diff
@@ -26,7 +26,7 @@ def run():
     args, bm_args = parser.parse_known_args()
 
     try:
-        benchmark = importlib.import_module(f"userbenchmark.{available_benchmarks[args.bm_name]}")
+        benchmark = importlib.import_module(f"userbenchmark.{available_benchmarks[args.bm_name]}.run")
```

Comment:
Hi @xuzhao9，seems `cpu` userbenchmark function was broken by this change, do we need to split the `__init__.py` into a `run.py`?